taichi/rhi/vulkan/vulkan_device.cpp

```diff
@@ -827,8 +858,11 @@ void VulkanCommandList::bind_pipeline(Pipeline *p) {
     return;
 
   if (pipeline->is_graphics()) {
```

Comment:
i wonder if we will have an way to opt out graphics support in the distributed runtime library in the future?

Reply:
Should be doable. (really doubt there will be significant size difference tho)